PROJECT

```diff
@@ -4,3 +4,12 @@ layout:
 projectName: odh-model-controller
 repo: github.com/opendatahub-io/odh-model-controller
 version: "3"
+resources:
+- api:
+    crdVersion: v1alpha1
+    namespaced: true
+  domain: opendatahub.io
+  group: nim
```

Comment:
```suggestion
  group: nim.opendatahub.io
```

Reply:
It seems using `nim` is ok, based on example from here: https://book.kubebuilder.io/cronjob-tutorial/basic-project.html

Reply:
The example you shared is for a CronJob, the group is _batch_, see [example](https://kubernetes.io/docs/concepts/workloads/controllers/cron-jobs/#example).

Take a look at the example from the same docs, diffrenet section, group is group.
https://book.kubebuilder.io/reference/project-config#example.

Reply:
But in practice, if in the PROJECT file, domain is [example.com](https://github.com/operator-framework/operator-sdk/blob/a268c70450f0743b29a603deb80bce1ab5c8bb74/testdata/go/v4/memcached-operator/PROJECT#L29) and group is [cache](https://github.com/operator-framework/operator-sdk/blob/a268c70450f0743b29a603deb80bce1ab5c8bb74/testdata/go/v4/memcached-operator/PROJECT#L30), the group of the resource will be [cache.example.com](https://github.com/operator-framework/operator-sdk/blob/a268c70450f0743b29a603deb80bce1ab5c8bb74/testdata/go/v4/memcached-operator/bundle/manifests/cache.example.com_memcacheds.yaml#L20C10-L20C27)?

Reply:
I don't think that's the way it works. Doing so will break the generation of bundle manifests, but I'm not sure it will affect boilerplate generation.

Reply:
You know what... I think you're right.